data-prepper-logstash-configuration/src/test/java/org/opensearch/dataprepper/logstash/mapping/DefaultLogstashPluginAttributesMapperTest.java

```diff
@@ -114,4 +115,20 @@ void mapAttributes_sets_additional_attributes_to_those_values() {
         assertThat(actualPluginSettings, hasKey(additionalAttributeName));
         assertThat(actualPluginSettings.get(additionalAttributeName), equalTo(additionalAttributeValue));
     }
+
+    @Test
```

Comment:
I would like to see an integration test added that contains the `opensearch` plugin.